napari/_qt/qt_dims_slider.py

```diff
@@ -40,9 +43,32 @@ def __init__(self, parent: QWidget, axis: int):
         self.axis = axis
         self.qt_dims = parent
         self.dims = parent.dims
-        self.label = None
+        self.axis_label = None
         self.slider = None
         self.play_button = None
+        self.curslice_label = QLineEdit(self)
```

Comment:
What do you think about adding a tooltip to the slice labels? I think this helps make it clear what they are when people are new to `napari`. It might be nice to do the same for the axis labels as well.

Reply:
The context for this comment is that while I like the spacing with the vertical separator between the current and total slice index elements, I think the vertical line is less intuitive than the slash.